content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -72,11 +72,19 @@ violated from the PodSecurity enforcement.
 See [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
 for more information.
 
+## apiserver.latency.k8s.io/total
+
+Example: `apiserver.latency.k8s.io/total: "573.018706ms"`
+
+This value shows total latency incurred in components within the apiserver. Please note that the latency is recorded in
+annotation only when the total latency of the given request exceeds 500ms; This is in keeping with the
+traces in rest/handlers for create, delete, update, get, list, and delete collection.
+
 ## authorization.k8s.io/decision
 
 Example: `authorization.k8s.io/decision: "forbid"`
 
-This annotation indicates whether or not a request was authorized in Kubernetes audit logs.
+Value must be **forbid** or **allow**. This annotation indicates whether or not a request was authorized in Kubernetes audit logs.
```

Comment:
This change does not align with the PR title and description. I think it belongs in a separate commit and possibly a different PR.

Reply:
Should i add a different PR? In case it is needed i'll add another PR for it.

Reply:
Two PRs is easiest.